scripts/dd/dd-compare.py

```diff
@@ -0,0 +1,160 @@
+# Purpose: Takes in two PRISSMM data dictionaries and print out a simple comparison report
+# Author: Sage Bionetworks
+# Date: 29AUG2024
+
+import pandas as pd
+import synapseclient
+import argparse
+from io import StringIO
+
+# Log in 
+syn = synapseclient.Synapse()
+syn.login()
+
+# Set the default PRISSMM data dictionaries
+
+default1 = syn.get('syn52903784').path #v4.0.1
+default2 = syn.get('syn61600728').path #v4.1.4
+
+# Initialize arg parser
+parser = argparse.ArgumentParser(description="Compare two data dictionaries.")
+parser.add_argument('--df1', type=str, default=default1, help='File path to the older data dictionary CSV')
+parser.add_argument('--df2', type=str, default=default2, help='File path to the newer data dictionary CSV')
+
+# Parse the arguments
+args = parser.parse_args()
+
+entity1 = syn.get(args.df1)
+entity2 = syn.get(args.df2)
+
+# Function to clean up the multiline content while preserving specific rows
```

Comment:
It might be helpful to provide an example of what said row looks like

Reply:
Provided an example within a docstring.